### PR TITLE
Add documentation

### LINKED ISSUE #90: Wiki entry on usage information
Add a radar-base wiki pages on how to use the connector
- [ ] how to use the webpage from clinician perspective.
- [ ] how to upload data directly to the backend from a data provider perspective.

### PR DESCRIPTION
**Pull request by nivemaham**
Adds documentation about the application and how to configure it

Fixes #90

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,182 @@
...
+A single sourceType entry is defined as below.
+
+```yaml
+
+- name: "acceleration-zip" # unique identifier of the data source or device (*required)
+    topics: # list of topics to send data
+      - "android_phone_acceleration" 
+    contentTypes: # content types of the data
+      - "application/zip"
+    timeRequired: false # if the data is large or if uploading data would require a lot of time mention this to true.
```
Comment (blootsvoets) [inline README.md]: This field was meant to indicate whether the user should supply a date-time for the upload, that is, that the uploaded content does not contain the relevant timestamp.

Comment (nivemaham) [reply README.md]: Oh I see. That makes sense. I will change it

**Comment by blootsvoets** [comment]
Could you please create PNG images instead of BMP?

I was thinking that the end-user documentation should be created in the radar-base.org wiki (#90) and the technical information be listed here (#81).

**Comment by nivemaham** [comment]
@blootsvoets Can we proceed with this now and come back to the developer documents later.

**Pull request merged**